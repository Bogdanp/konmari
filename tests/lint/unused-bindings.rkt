#lang racket/base

(provide
 h)

(define (f x)
  (define y 0)
  1)

(f 1)

(define ((g x) y)
  x)

(define (h)
  (displayln "I'm provided!"))

(let ([x 1]
      [y x])
  y)

(let ([x 1]
      [y 2])
  3)

(define (i #:ok? ok?)
  #f)

(define (j #:ok? [ok? #f])
  #t)

(cond
  [#t
   (define z 1)
   #f])
