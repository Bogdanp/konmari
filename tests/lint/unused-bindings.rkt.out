/unused-bindings.rkt:7:11:warning:identifier 'y' is never used
/unused-bindings.rkt:6:12:warning:identifier 'x' is never used
/unused-bindings.rkt:12:16:warning:identifier 'y' is never used
/unused-bindings.rkt:23:8:warning:identifier 'y' is never used
/unused-bindings.rkt:22:8:warning:identifier 'x' is never used
/unused-bindings.rkt:12:11:warning:identifier 'g' is never used
